README.md

```diff
@@ -58,7 +58,7 @@ Alternatively you can specify an alias in your `$HOME/.clojure/deps.edn`
 And then run with a simpler:
 
 ```shell
-$ clojure -A:rebel
+$ clojure -M:rebel
```

Comment:
Removing the `$` would be great too: Github's new copy button automatically includes it.

```suggestion
clojure -M:rebel
```